src/mavsdk/plugins/camera/camera_impl.cpp

```diff
@@ -222,6 +225,7 @@ void CameraImpl::manual_disable()
 {
     invalidate_params();
     _parent->remove_call_every(_camera_information_call_every_cookie);
+    // TODO: should this remove_call_every for _status.call_every_cookie?
```

Comment:
@julianoes: not sure about what `manual_disable()` does... should it `remove_call_every(_status.call_every_cookie)` here?

Reply:
It's used when you select another camera. Not sure, I'd probably add it for consistency.

Reply:
Up to you, otherwise this is fine.

Reply:
Oh right, it should otherwise it will leak a callback when `manual_enable()` is called again :thinking:. I'll change that!